CHANGELOG.md

```diff
@@ -1,3 +1,146 @@
+# v1.13.0 (2022-03-15)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to Bottlerocket ([#2829])
+* Create and set-up data partition at runtime ([#2807], [#2879], [#2873])
```

Comment:
Sorry, on second thought "create" isn't accurate since we do create the partition in `rpm2img`.
```suggestion
* Set-up data partition at runtime ([#2807], [#2879], [#2873])
```